src/components/HomeSearchBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const [searchItem, setSearch] = useState('')
+
+  const handleClickSearch = (e) => {
+    window.open(`products/${searchItem}`)
```

Comment:
Just a question, do we really need to open a new tab when clicking a search icon? If not I would highly recommend using [next/router](https://nextjs.org/docs/api-reference/next/router) instead

Reply:
I way I see things is that users will have our home page still placed on their `tab` when they're finished browsing a particular product. What do you think?

Reply:
Just for demonstration, could you provide me some example websites please?

Reply:
My bad, I thought we have done a searching thing in just a home search. I accept your thought and will apply it then ka. 🙈